server/sonar-web/src/main/js/apps/overview/meta/MetaTagsSelector.js

```diff
@@ -42,7 +40,7 @@ type State = {
 
 const LIST_SIZE = 10;
 
-class ProjectTagsSelectorContainer extends React.PureComponent {
+export default class MetaTagsSelector extends React.PureComponent {
```

Comment:
I would keep the word "Project" in the name since it is not generic and will only work for projects tags.